src/ClientData/DataManagerTest.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <thread>
+#include <vector>
+
+#include "ClientData/DataManager.h"
+#include "OrbitBase/ThreadUtils.h"
+
+namespace orbit_client_protos {
+bool operator==(const CallstackEvent& lhs, const CallstackEvent& rhs) {
+  return lhs.thread_id() == rhs.thread_id() && lhs.callstack_id() == rhs.callstack_id();
```

Comment:
This is dangerous. If someone else also defines the operator in a different cpp file we might end up with an ODR violation. I think there are a couple of options:

1. Mark the operator static to get internal linkage.
2. Use `MessageDifference::Equals` (https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.util.message_differencer#MessageDifferencer.Equals)

Reply:
Ah right, I missed that, thanks a lot. I changed this to `static`.